doc/units.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+.. _units:
+
+Internal representation (units)
+===============================
+
+When importing data, MNE-Python will always convert measurements to the same
+standard units. Thus the in-memory representation of data are always in:
+
+- Volts (eeg, eog, seeg, emg, ecg, bio, ecog)
+- Teslas (magnetometers)
```

Comment:
I think the plural of Tesla should be Tesla. Willing to entertain a counter example if you can find one ;-)

Reply:
From [Wikipedia page Tesla_(unit)](https://en.wikipedia.org/wiki/Tesla_(unit)):

> The unit of electric field in the MKS system of units is newtons per coulomb, N/C, while the magnetic field (in **teslas**) can be written as N/(C·m/s).

(emphasis added)